src/styles/getMuiTheme.js

```diff
@@ -224,4 +226,12 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  if (computedTheme.isRtl) {
```

Comment:
> after rtl is applied?

Actually technically the issue would be if they wanted transformers to run *before* rtl is applied. If we push at the end of the list as the current implementation is doing it, it will be the first one ran, then custom transformers (since we're using lodash's `flowRight` instead of `flow`)